src/components/Brands/Brands.js

```diff
@@ -0,0 +1,24 @@
+import { categories } from "../../backend/db/categories";
+
```

Comment:
you should fetch the categories using a API call instead of importing directly from backend folder

Reply:
okk. Done with api call of categories. Thanks for your suggestion